### PR TITLE
Fix @ckeditor/ckeditor5-engine getAttribute

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://ckeditor.com/docs/ckeditor5/latest/api/module_engine_model_node-Node.html#function-getAttribute
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ckeditor__ckeditor5-engine ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ckeditor__ckeditor5-engine@latest/))
was missing the following properties:
1. getBoxSidesValues
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 01c0c6f223e987c93ab24148b6361d90abc0a2c5
</p>

File: types/ckeditor__ckeditor5-engine/src/model/node.d.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export default class Node {
 
     constructor(attrs?: Parameters<typeof toMap>[0]);
     getAncestors(options?: { includeSelf?: boolean | undefined; parentFirst?: boolean | undefined }): Array<Element | DocumentFragment>;
-    getAttribute(key: string): string | undefined;
+    getAttribute(key: string): string | number | boolean | undefined;
```
Comment (peterblazejewicz) [inline types/ckeditor__ckeditor5-engine/src/model/node.d.ts:30]: are those really returning something different compared to native implementation (a string or a null)

Comment (fedemp) [reply types/ckeditor__ckeditor5-engine/src/model/node.d.ts:30]: Yes, cause attributes for the model are saved in a `Map`.

Screenshot from 2022-03-19 20-10-08
Screenshot from 2022-03-19 20-16-15

Comment (peterblazejewicz) [reply types/ckeditor__ckeditor5-engine/src/model/node.d.ts:30]: txh!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**